spec/controllers/hunters_improvements_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
   # HuntersImprovementsController. Be sure to keep this updated too.
   let(:valid_session) { {} }
 
-  before(:each) do
+  before do
```

Comment:
Was this a linter fix? If so, in another PR, I'd like to configure the Rubocop rules to recommend the more specific option.

Reply:
It was a local issue that I have fixed.

Reply:
Resolved with commit d1c055e.